docs/cases/api/cases-api/cases-api-find-cases-by-alert.asciidoc

```diff
@@ -42,11 +42,20 @@ GET api/cases/alerts/293f1bc0-74f6-11ea-b83a-553aecdb28b6
 
 ==== Response payload
 
-A JSON array listing the retrieved case IDs.
+A JSON array containing case IDs associated with the specified alert. Case titles are also retrieved.
```

Comment:
```suggestion
A JSON array containing any case IDs associated with the specified alert, and case titles.
```

Reply:
Amendment to avoid a comma splice:
```suggestion
A JSON array containing any case IDs and case titles associated with the specified alert.
```

Reply:
`"and case titles"` sounds a tad odd. Let's leave the original for now but +1 for the first suggestion.

Reply:
Hm, I'm not entirely sure whether the alert ID is associated with the case title. @cnasikas what are your thoughts on the recommended revision? Is it an accurate statement?

Reply:
I think if we mention the cases IDs then we should mention the cases titles. The alert ID is not associated with the title. The case ID is associated with the title. The response contains all cases that the alert is attached. From the cases properties we choose to return the id and the title.

Reply:
Thanks, @cnasikas ! I'll leave the edit as it is then.